reconcile/utils/ocm.py

```diff
@@ -931,11 +931,37 @@ def _init_blocked_versions(self, blocked_versions):
                 raise TypeError(f"blocked version is not a valid regex expression: {b}")
 
     @retry(max_attempts=10)
-    def _get_json(self, api):
-        r = requests.get(f"{self.url}{api}", headers=self.headers)
+    def _do_get_request(self, api: str, params: dict[str, str]):
+        r = requests.get(f"{self.url}{api}", headers=self.headers, params=params)
         r.raise_for_status()
         return r.json()
 
+    @staticmethod
+    def _response_is_list(rs: dict[str, Any]) -> bool:
+        return rs["kind"].endswith("List")
+
+    def _get_json(self, api):
+        responses = []
+        params = {"size": 100}
+        while True:
+            rs = self._do_get_request(api, params=params)
```

Comment:
Just curious if this will use keep-alive and how much that matters? ([docs](https://docs.python-requests.org/en/latest/user/advanced/#keep-alive))

Reply:
See timeout